charts/argo-workflows/values.yaml

```diff
@@ -257,6 +257,14 @@ server:
       #   backend:
       #     serviceName: ssl-redirect
       #     servicePort: use-annotation
+      ## for Kubernetes >=1.19 (when "networking.k8s.io/v1" is used)
+      # - path: /*
+      #   pathType: Prefix
```

Comment:
```suggestion
```
Setting `pathType` via values is not supported (yet).

Reply:
looks like it IS supported for `extraPaths` because the value is copied as is from values to Ingress

Please take a look in https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/templates/argocd-server/ingress.yaml#L33-L35

```yaml
    - host: {{ $host }}
      http:
        paths:
          {{- if $extraPaths }}
          {{- toYaml $extraPaths | nindent 10 }}
          {{- end }}
```

Reply:
Ah yes 👍 you are right